### PR TITLE
Change design and responsive for form footer

### PR DESCRIPTION
**Pull request by Morel-Victor**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
ok for me :+1:

File: src/Components/Form/index.jsx [hunk 1]
```diff
@@ -101,11 +99,10 @@ export default function Form() {
             chooseOption === 'advice'
           }
         />
-
         <input className="comments" type="search" placeholder="Commentaire" />
       </form>
       <Button>
-        <input type="submit" value="Envoyer" />
+        <p>Envoyer</p>
```
Comment (Hacene010) [inline src/Components/Form/index.jsx]: Why did you add a p tag here ?

**Review by Hacene010 — CHANGES_REQUESTED** [review]

File: src/Components/Form/index.jsx (hunk 1 above)
Comment (Morel-Victor) [reply src/Components/Form/index.jsx]: Done !

**Review by Caroline-CD — DISMISSED** [review]
You forgot to make the requested changes !

**Review by Caroline-CD — APPROVED** [review]
ok for me :)

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**